scripts/python/SLiM_sims.py

```diff
@@ -6,32 +6,35 @@
 
 import subprocess
 import sys
+import os
 
 
-def parse(out):
-    lines = out.split("\n")
-    for line in lines:
-        print(line)
-        print("--------------")
+def run_slim(N, bot):
 
-
-def main():
-
-    N = int(sys.argv[1])  # initial N value
-    bottleneck = float(sys.argv[2])  # proportion of N
+    # Change directory to script path so it can be run from any directory in project
+    os.chdir(sys.path[0])
 
     # Call SLiM from command line with N and bottleneck proportion values
     # (passed as command-line arguments)
-    process = subprocess.Popen(["slim", "-d", "N=" + str(N), "-d", "bottleneck=" + str(bottleneck), "../slim/Bottleneck.slim"],
+    process = subprocess.Popen(["slim", "-d", "N=" + str(N), "-d", "bot=" + str(bot), "../slim/Bottleneck.slim"],
                                stdout = subprocess.PIPE,
                                stderr = subprocess.PIPE,
                                universal_newlines = True)
     out, err = process.communicate()
 
     # out is output of slim
     # err is error message from slim
-    parse(out)  # calls function to parse the output
+    # print(out)  # calls function to parse the output
     print(err)  # prints error message
 
+
 if __name__ == "__main__":
-    main()
+    import argparse
+    parser = argparse.ArgumentParser()
+    parser.add_argument("N", help = "The desired population size", type = int)
+    parser.add_argument("bot", help = "The desired strength of the population bottleneck. Expressed as the proportion of the population sampled during the bottleneck. 1.0 = No bottleneck", type = float)
```

Comment:
I'm being pedantic as hell, but given that this PR *is* about cleaning up scripts, we should get rid of the spaces around `=` bc PEP8

Reply:
Done. For the record, this is one of those PEP8 requirements that I don't fully agree with. I find it makes the code a little less readable.